src/components/Buttons/ShortcutButtons.tsx

```diff
@@ -73,31 +73,31 @@ const ShortcutButtons = ({
   const lockWallet = () => {
     dispatch(walletLocked())
 
-    posthog?.capture('Locked wallet', { origin: analyticsOrigin })
+    posthog.capture('Locked wallet', { origin: analyticsOrigin })
```

Comment:
Need to check, but isn't there another "Locked wallet" event somewhere, not necessarily attached to the shortcut button? 

What would be interesting here I believe, is to identify which shortcut button is clicked. Ie. "is it the one on the home page"? 

Same for the events below. WDYT?

Reply:
This is already the case :) 

![image](https://github.com/alephium/desktop-wallet/assets/1579899/b0ac03f1-07e7-49b5-b8a2-f550a5814114)